benchmarks/experimental/benchmark_dataset.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+
+import torch
+from torch.utils.data import Dataset
+
+
```

Comment:
Can we add a TODO that this file will be removed?

Reply:
Done